cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ExecutableType.java

```diff
@@ -0,0 +1,11 @@
+package io.cloudslang.lang.entities;
+
+/**
+ * User: bancl
+ * Date: 1/6/2016
+ */
+public enum ExecutableType {
```

Comment:
we need to add a dependency to cloudslang-runtime in the cloudslang-compiler pom to do that.